go-controller/pkg/ovn/ovn.go

```diff
@@ -104,7 +119,13 @@ func (oc *Controller) WatchEndpoints() {
 				logrus.Errorf("Error in adding load balancer: %v", err)
 			}
 		},
-		UpdateFunc: func(old, new interface{}) { return },
+		UpdateFunc: func(old, new interface{}) {
+			ep := new.(*kapi.Endpoints)
+			err := oc.addEndpoints(ep)
```

Comment:
@rajatchopra That makes sense. We need a deepequal here too, right? Or are there other golang/k8s tricks to compare new and old objects?